docs/content/what-is-a-temporal-cron-job.md

```diff
@@ -67,10 +68,25 @@ Use the [crontab guru site](https://crontab.guru/) to test your cron expressions
 
 **`robfig` predefined schedules**
 
-You can also pass any of the [predefined schedules](https://pkg.go.dev/github.com/robfig/cron#hdr-Predefined_schedules) or [intervals](https://pkg.go.dev/github.com/robfig/cron#hdr-Intervals) described in the [`robfig/cron` documentation](https://pkg.go.dev/github.com/robfig/cron).
+You can also pass any of the [predefined schedules](https://pkg.go.dev/github.com/robfig/cron/v3#hdr-Predefined_schedules) or [intervals](https://pkg.go.dev/github.com/robfig/cron/v3#hdr-Intervals) described in the [`robfig/cron` documentation](https://pkg.go.dev/github.com/robfig/cron/v3).
 
 For example, "@weekly" causes a Workflow Execution to spawn once a week at midnight between Saturday and Sunday.
 
+**Time zones** (in Temporal 1.15)
+
+You can change the time zone that a Cron Schedule is interpreted in by prefixing the specification with `CRON_TZ=America/New_York ` (or your desired time zone).
+
+:::warning
+
+Using time zones in production introduces a surprising amount of complexity and failure modes! If at all possible, we recommend specifying Cron Schedules in UTC (the default). If you need to use time zones, here are a few edge cases to keep in mind:
+
+- If a cron job is scheduled around the time when daylight saving time begins or ends, e.g. `30 2 * * *`, it may run zero, one, or two times in a day! The Cron library that we use does not do any special handling of DST transitions. Avoid schedules that include times in DST transitions.
```

Comment:
```suggestion
- If a Temporal Cron Job is scheduled around the time when daylight saving time begins or ends, e.g. `30 2 * * *`, it may run zero, one, or two times in a day! The Cron library that we use does not do any special handling of DST transitions. Avoid schedules that include times in DST transitions.
```